common/djangoapps/enrollment/views.py

```diff
@@ -788,6 +789,7 @@ class CourseEnrollmentsApiListView(DeveloperErrorViewMixin, ListAPIView):
     permission_classes = IsAdminUser,
     throttle_classes = EnrollmentUserThrottle,
     serializer_class = CourseEnrollmentsApiListSerializer
+    pagination_class = CourseEnrollmentsApiListPagination
```

Comment:
Need to update the docstring -- this pagination class means the `page_size` querystring parameter is ignored, unlike with the [default pagination class](https://github.com/edx-solutions/edx-platform/blob/development/openedx/core/lib/api/paginators.py#L9).

Reply:
@jill, the version of `djangorestframework` that is used doesn't support `page_size` querystring parameter, but newer versions do. Will remove it from the docstring. :+1:

Reply:
Ah k!  Thanks.

Reply:
Addressed in f1cae9b0a610bd75713723e4a228762e77779378